ceph.spec.in

```diff
@@ -379,6 +379,15 @@ developed as part of the Ceph distributed storage system. This is a
 shared library allowing applications to access the distributed object
 store using a simple file-like interface.
 
+%package -n ceph-plugins-devel
```

Comment:
Another minor nit: you could drop the "-n ceph-" here - without "-n" it'll automatically pickup the "ceph-" prefix.